pages/common/gcloud.md

```diff
@@ -0,0 +1,32 @@
+# gcloud
+
+> The official CLI tool for Google Cloud Platform. You can use this tool to perform many common platform tasks either from the command-line or in scripts and other automations.
+> Commands can be run using the _gcloud_ command.
+
+- List all properties in your active configuration. Including: current authorized account, current project, and the default region and zone, if set.:
```

Comment:
Remove the full stop before the colon here.

Reply:
SGTM